### PR TITLE
Add rmsip docs

### PR DESCRIPTION
**Pull request by lilyminium**
~~Fixes #~~

Changes made in this Pull Request:
 - Add an example to RMSIP function to clarify use

PR Checklist
------------
 - [ ] Tests?
 - [x] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

**Comment by lilyminium** [comment]
RTD is throwing an error on installing openblas, but `make html` locally produces the following:

Screen Shot 2022-10-18 at 12 35 23 pm
Screen Shot 2022-10-18 at 12 35 40 pm

File: package/doc/sphinx/source/conf.py [hunk 1]
```diff
@@ -39,12 +39,18 @@
...
+    'sphinx.ext.autodoc',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.mathjax',
+    'sphinx.ext.viewcode',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.todo',
+    'sphinx_sitemap',
+    'sphinx_rtd_theme',
+    'sphinxcontrib.bibtex',
+    'sphinx.ext.doctest',
```
Comment (IAlibay) [inline package/doc/sphinx/source/conf.py:52]: new entries?

**Review by IAlibay — APPROVED** [review]

**Comment by IAlibay** [comment]
The RTD openblas error is prevalent on more or less every conda build atm (I think mamba builds are not affected because they use use cf as the base channel), I suspect it'll get fixed in the morning. However it shouldn't be a blocker to merging this imho.

File: package/doc/sphinx/source/conf.py (hunk 1 above)
Comment (lilyminium) [reply package/doc/sphinx/source/conf.py:52]: I added `doctest` but everything else is just a reformat

**Pull request merged**